packages/edit-site/src/components/block-editor/use-site-editor-settings.js

```diff
@@ -144,7 +139,6 @@ export function useSpecificEditorSettings() {
 
 			richEditingEnabled: true,
 			supportsTemplateMode: true,
-			focusMode: canvasMode === 'view' && focusMode ? false : focusMode,
```

Comment:
This is a breaking change; the spotlight mode will now apply in view mode.

Reply:
I can restore it but I don't like that part :P

Reply:
I think the current behavior creates bad UX when exploring theme styles.

## Screenshot

https://github.com/WordPress/gutenberg/assets/240569/355871aa-9ed6-4f57-b48f-3e349aa2f023